trymito.io/pages/mito_pro_roadmap.tsx

```diff
@@ -0,0 +1,158 @@
+                        </h1>
+                        <p>
+                            Import custom python snippets to use within the Mito Spreadsheet and add completely new step types to the Mito Toolbar.  
+                        </p>
+                    </div>
+                    <div className={securityStyles.security_bullet_container + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={FlagIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Graph <br/> styling
+                        </h1>
+                        <p>
+                            Create presentation-ready graphs with full control of colors, ability to save graph templates, and more. 
+                        </p>
+                    </div>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={FlagIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                        Connect to any data source
+                        </h1>
+                        <p>
+                            Connect to databases and remote file systems so users can import any data set  without having to write custom pandas code.   
+                        </p>
+                    </div>
+                    <div className={securityStyles.security_bullet_container + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={FlagIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Feature <br/> settings
+                        </h1>
+                        <p>
+                            Customize Mito by turning on/off code optimization, code documentation, seleting between light and dark mode, and more. 
+                        </p>
+                    </div>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={FlagIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Advanced <br/> formatting
+                        </h1>
+                        <p>
+                            Utilize Excel-like formatting and conditional formatting your analysis stand out. And generate Python code for all of it.   
```

Comment:
And generate Python code for all of it. -- maybe not -- we don't know if we can do this... maybe we can just say "preserve this formatting when you export"?